### ISSUE TITLE
Change pre-trained model code

### ISSUE THREAD

**Issue by ahsan44411** [facebookresearch/detectron2#584]
## ❓ How to use Detectron2

Where to change the code in the pre trained models so that it outputs F-measure along along with the loss that it computes.
